packages/bugsnag_flutter/lib/src/model/device.dart

```diff
@@ -44,10 +73,26 @@ class Device {
       };
 }
 
+/// Stateful information set by the notifier about the device on which the event
+/// occurred can be found on this class. These values can be accessed and
+/// amended if necessary.
+///
+/// See also:
+/// - [Device]
+/// - [BugsnagEvent]
+/// - [App]
 class DeviceWithState extends Device {
+  /// The number of free bytes of storage available on the device
   int? freeDisk;
+
+  /// The number of free bytes of memory available on the device
   int? freeMemory;
+
+  /// The orientation of the device when the event occurred: either
```

Comment:
```suggestion
  /// The orientation of the device when the event occurred: e.g.
```

There are more possible values than the two listed 😂